overlay_gl/overlay.c

```diff
@@ -153,16 +153,18 @@ static void newContext(Context *ctx) {
 		}
 	}
 
-	char *xdgRuntimeDir = getenv("XDG_RUNTIME_DIR");
+	char *xdgRuntimeDir      = getenv("XDG_RUNTIME_DIR");
+	char *overlayPipeXdgDir  = "MumbleOverlayPipe";
+	char *overlayPipeHomeDir = ".MumbleOverlayPipe";
 
-	if (xdgRuntimeDir != NULL) {
+	if (xdgRuntimeDir != NULL && strlen(xdgRuntimeDir) + strlen(overlayPipeXdgDir) < sizeof(ctx->saName.sun_path)) {
```

Comment:
```suggestion
	if (xdgRuntimeDir != NULL && strlen(xdgRuntimeDir) + strlen(overlayPipeXdgDir) < strlen(ctx->saName.sun_path)) {
```
`sizeof` would only give use the size in bytes of the underlying type, which should always return 8 bytes since `sun_path` is a (`char`) pointer. However, we need to know the size of the underlying buffer.